var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -13,7 +13,7 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='main')
 
-    version('4.4.36', commit='07fe9999a137c3741fd95713149a76a202cb7d7a')
+    version('4.5.30', commit='3fdb48cad0339cb2eb36d1ad4a7004983525af6a')
 
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('neuron+python', type='run')
```

Comment:
In https://bbpgitlab.epfl.ch/cells/bglibpy/-/blob/df06056e7102380905150d3651969d5e59d9625a/setup.py#L20 there is also a restriction to `pandas<1.3.0`, but according to the linked ticket BLPY-267 it shouldn't be needed anymore so I think it's correct to not include it here, right? (in that case it should be removed in setup.py, but it's unrelated to this PR).